initial-data-load/README.md

```diff
@@ -0,0 +1,27 @@
+| LOG_LABEL                          | no       | oscar-initial-data-upload                                          | Title of the logs that you can see in the terminal when you run the service.                                                                                                                                                                                                                                     |
+| AZURE_STORAGE_CONNECTION_STRING    | yes      | -                                                                  | Default Azurite connection string from documentation used for a local development.                                                                                                                                                                                                                               |
+| AZURE_STORAGE_LAYER_CONTAINER_NAME | yes      | layer-geo-data                                                     | Name of the Blob Container to which geojsons are saved in the Azurite.                                             |
+| MONGO_URI                          | yes      | mongodb://localhost:27917/oscar or mongodb://localhost:27017/oscar | URI with port on which the MongoDB is running. In the development environment it runs on port 27917 and in the testing environment on port 27017.                                                                                                                                                                |
+| UPLOAD_DATA_TYPES                  | no       | -                                                                  | Data sets to be uploaded into the database. There are two options: 'mapLayers' for the map view configuration and 'attributes' for application data.                                                                                                                                                             |
+| COUNTRY                            | yes      | ''                                                                 | Name of the folder from which you get the data for the specific country.                                                                                                                                                                                                                                         |
+| NEW_STORAGE_CONTAINERS             | no       | ''                                                                 | Names of the Blob Containers in the Azurite.                                                                                                                                                                                                                                                                     |
```

Comment:
Names of the Blob Containers to be created in Azure Storage.

Reply:
I fixed this.